src/app/shared/interfaces/http-user.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { HttpUserName } from "./http-user-name.interface";
+
+export interface HttpUser {
```

Comment:
This interface can extent the Token and Login interface
Please remove the dublication

Reply:
ok, done